lib/datasource/index.spec.ts

```diff
@@ -34,19 +34,27 @@ describe(getName(), () => {
     expect(datasource.getDatasources()).toBeDefined();
     expect(datasource.getDatasourceList()).toBeDefined();
   });
-  it('validates dataource', () => {
+  it('validates datasource', () => {
     function validateDatasource(module: DatasourceApi, name: string): boolean {
       if (!module.getReleases) {
         return false;
       }
-      if (module.id !== name) {
-        return false;
-      }
-      return true;
+      return module.id === name;
+    }
+    function filterClassBasedDatasources(name: string): boolean {
+      return !['cdnjs', 'clojure'].includes(name);
     }
     const dss = datasource.getDatasources();
```

Comment:
dss is a `Map`, so clone like this:

```suggestion
    const dss = new Map(datasource.getDatasources());
```

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map/Map